.gitmodules

```diff
@@ -1,4 +1,4 @@
 [submodule "docker/schemas"]
 	path = schemas
 	url = git@github.com:Shared-Reality-Lab/IMAGE-server.git
-	branch = schemas
+	branch = ner-image
```

Comment:
Should not be changing submodule branch! Merge should be made from ner-image into schemas *before* this gets merged!